applications/CoSimulationApplication/python_scripts/co_simulation_steady_analysis.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import print_function, absolute_import, division  # makes these scripts backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics as KM
+from KratosMultiphysics.analysis_stage import AnalysisStage
+
+# CoSimulation imports
+from KratosMultiphysics.CoSimulationApplication.co_simulation_analysis import CoSimulationAnalysis
+# Other imports
+import sys
+
+class CoSimulationSteadyAnalysis(CoSimulationAnalysis):
```

Comment:
please remove this class

=> define a dummy `Predict` and `AdvanceInTime` in the `PotentialFlowWrapper`
@inigolcanalejo told me he will implement those methods soon, this will make it easier to update